Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -167,10 +167,10 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
 
   obj.AddAction(
          "AddForceXY",
-         _("Add a force"),
-         _("Add a force to an object. The object will move according to "
-           "all of the forces it has."),
-         _("Add to _PARAM0_ _PARAM3_ force of _PARAM1_ p/s on X axis and "
+         _("Add a velocity ("force")"),
```

Comment:
```suggestion
         _("Add a velocity (Force)"),
```
For fit with current format in app.
![image](https://user-images.githubusercontent.com/1670670/121664118-6eab4780-caa7-11eb-96c0-e6e8a74485dc.png)